samples/unpeelingOnion/src/core/validation.ts

```diff
@@ -0,0 +1,81 @@
+//////////////////////////////////////
+/// Validation
+//////////////////////////////////////
+
+export const enum ValidationErrors {
+  NOT_A_NONEMPTY_STRING = 'NOT_A_NONEMPTY_STRING',
```

Comment:
Enum value should be lower letters -> easier to create an error message.

Reply:
Thank you, I’m aware of this convention, but my personal preference is to have it more expressive 🙂